packages/survey-creator-core/src/creator-base.ts

```diff
@@ -2792,7 +2792,7 @@ export class CreatorBase extends Base
     if (obj["questions"]) {
       obj["questions"].forEach(q => this.updateConditionsOnRemove(q));
     }
-    obj["delete"]();
+    obj["removeFromParent"]();
```

Comment:
Please use obj["delete"](false). Here is [the commit](https://github.com/surveyjs/survey-library/commit/b544af8b1e2035c14c29e32a12efe721d9dc509b),
Thank you,
Andrew